locales-pending/how-it-works.ftl

```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## Banner
+
+how-it-works-banner-title = How it works
+how-it-works-banner-text = We help fix exposures of your information from two different sources: <data_brokers_link>data brokers</data_brokers_link> and <data_breaches_link>data breaches</data_breaches_link>.
+
+## Section 1 - We remove your info from data brokers
+## Intro
+
+section-1-intro-title = We remove your info from <data_brokers>data brokers</data_brokers>
+section-1-intro-text-1 = These sites scrape your data from across the web to create profiles that include your name(s), current and previous addresses, family member names, criminal history, and much more. And they sell this information to anyone searching for you.
+section-1-intro-text-2 = Getting your information off these sites helps protect your privacy and keep stalkers or harassers from finding info like your phone number or home address.
+section-1-intro-cta-button = Get data removal
+
+## Step 1
+
+section-1-step-1-title = Step 1
+section-1-step-1-subtitle = We scan for your private information using only your:
+section-1-step-1-name = <name>Name:</name> Your legal name will help us find profiles of you, but aliases or previous names may also appear in search results.
+section-1-step-1-location = <location>Location:</location> Your current city and state will help us narrow down the results, but past addresses may also show up.
+section-1-step-1-dob = <dob>Date of birth:</dob> This will reduce chances of finding profiles of people with the same name as you.
+section-1-step-1-closing-text = In seconds, we’ll show you how many sites have made profiles of you, and exactly what information they are selling.
+
+## Step 2
+
+section-1-step-2-title = Step 2
+section-1-step-2-subtitle = We automatically remove your information
+section-1-step-2-text-1 = Each data broker needs to be contacted to begin the data removal process, which takes time and effort.
+section-1-step-2-text-2 = For paid subscribers, as soon as we identify which sites have your information, we automatically initiate a request for removal. Free members will be alerted which data brokers to contact themselves.
+
+## Step 3
+
+section-1-step-3-title = Step 3
+section-1-step-3-subtitle = We watch for re-exposures, and remove again
+section-1-step-3-text-1 = Sites that sell your data continually scrape new information about you on sources across the web — and they will often re-add you to their sites, even if you’ve removed it.
+section-1-step-3-text-2 = For paid subscribers, we re-check every month to make sure they don’t add you back. It’s an ongoing process and you can rest assured that no matter how many times your name reappears on their site, we will always work to remove it.
+
+## CTA button
```

Comment:
This should be a single dash comment, no empty line after, since it refers to one string.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/612b51da9ca51575a9b9b8150c3dbabe627f4c63